backend/deepchecks_monitoring/api/v1/check.py

```diff
@@ -258,7 +258,23 @@ async def get_checks(
     await Model.fetch_or_403(session, model.id, user)
     q = select(Check).join(Check.model).where(model_identifier.as_expression)
     results = (await session.scalars(q)).all()
-    return [CheckSchema.from_orm(res) for res in results]
+    check_schemas = [CheckSchema.from_orm(res) for res in results]
+
+    def get_check_order(check):
+        order_by_name = ['SingleDatasetPerformance',
```

Comment:
if you can easily check if labels are available and if not put LabelDrift and SingleDatasetPerformance last - would be great